seata-spring-boot-starter/src/main/java/io/seata/spring/boot/autoconfigure/provider/SpringBootConfigurationProvider.java

```diff
@@ -114,42 +125,46 @@ private Object get(String dataId) throws IllegalAccessException {
      */
     private String convertDataId(String rawDataId) {
         if (rawDataId.startsWith(SPECIAL_KEY_VGROUP_MAPPING)) {
-            return StarterConstants.SERVICE_PREFIX + "." + NORMALIZED_KEY_VGROUP_MAPPING;
+            return SERVICE_PREFIX + DOT + NORMALIZED_KEY_VGROUP_MAPPING;
         }
         if (rawDataId.endsWith(SPECIAL_KEY_GROUPLIST)) {
-            return StarterConstants.SERVICE_PREFIX + "." + NORMALIZED_KEY_GROUPLIST;
+            return SERVICE_PREFIX + DOT + NORMALIZED_KEY_GROUPLIST;
         }
         if (rawDataId.startsWith(SPECIAL_KEY_UNDO)) {
             String suffix = StringUtils.removeStart(rawDataId, SPECIAL_KEY_UNDO);
-            return StarterConstants.UNDO_PREFIX + "." + StringFormatUtils.dotToCamel(suffix);
+            return UNDO_PREFIX + DOT + StringFormatUtils.dotToCamel(suffix);
         }
         if (rawDataId.startsWith(SPECIAL_KEY_CLIENT_LOCK)) {
             String suffix = StringUtils.removeStart(rawDataId, SPECIAL_KEY_CLIENT_LOCK);
-            return StarterConstants.LOCK_PREFIX + "." + StringFormatUtils.minusToCamel(
+            return LOCK_PREFIX + DOT + StringFormatUtils.minusToCamel(
                 StringFormatUtils.dotToCamel(suffix));
         }
-        if (rawDataId.startsWith(SPECIAL_KEY_CLIENT)) {
-            String suffix = StringUtils.removeStart(rawDataId, SPECIAL_KEY_CLIENT);
-            return StarterConstants.CLIENT_PREFIX + "." + StringFormatUtils.dotToCamel(suffix);
+        if (rawDataId.startsWith(SPECIAL_KEY_CLIENT_RM)) {
+            String suffix = StringUtils.removeStart(rawDataId, SPECIAL_KEY_CLIENT_RM);
+            return CLIENT_RM_PREFIX + DOT + StringFormatUtils.dotToCamel(suffix);
```

Comment:
minusToCamel?

Reply:
lock {
      retry.internal = 10
      retry.times = 30
      retry.policy.branch-rollback-on-conflict = true
    }
Yes.For retry.policy.branch-rollback-on-conflict = true.will need minusToCamel conversion.

Reply:
![image](https://user-images.githubusercontent.com/8758457/73595130-d9618000-454f-11ea-99d5-97e6185b2bbc.png)
Will the `hyphen` code style change to `camel`?

Reply:
Spring-boot will do the conversion automatically.

Reply:
minusToCamel is used to convert the items read from original configuration.

Reply:
In the code is actually read the dot between configuration items, I wrong thought is a hyphen interval.